### PR TITLE
General bug fix and some QoL features

### PR DESCRIPTION
**Pull request by TUsama**
It can't be merged rn cuz this PR uses some code from my last PR, after the ExileTooltips PR is merged I'll solve the conflict.

### CODE REVIEW

**Comment by RobertSkalko** [comment]
I'm going to be fixing some stuff from the last PR and tweaking a bit. Make sure this is conflict free and doesn't change my edits

**Comment by RobertSkalko** [comment]
and tell me when this one is ready for merging btw

**Comment by TUsama** [comment]
sure, currently I'm playing a new run of CTE2 so it might take some time.

File: src/main/java/com/robertx22/age_of_exile/uncommon/localization/Command.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
...and create a new team."), CREATE_WHEN_IN_A_TEAM("You are already in a team, do you still want to create a new team? "), CANT_MAKE_LEADER("You are already the team leader."), LEAVE_TEAM("You have leaved the team."), AUTO_PVE_TEAMMATE(" (Auto PVE)"), NOT_IN_SAME_TEAM_DUE_TO_DISTANCE(" (Too Far)"), CANT_INVITE_YOURSELF("you can't invite yourself!"), NOT_IN_A_TEAM_CURRENTLY("Player isn't in a team"), NOT_INVITED("You aren't invited to their team"), JOIN_TEAM("You have joined %1$s's team."), KICK_INFO("You have kicked %1$s from your team."), BEEN_KICKED("You have been kicked from your team."), YOU_ARE_NOT_LEADER("you aren't the leader."), YOU_ARE_LEADER("You have been made a team leader."), TRANSFER_LEADERSHIP("%1$s is now a team leader."), PLAYER_NOT_IN_TEAM("They aren't on your team"), CLICK_TO_CREATE("You can click here to create a team!"), INVITE_INFO("You have invited %1$s to join your team."), CLICK_TO_JOIN("You can also click here to join!"), JOIN_TIP("Do /mine_and_slash teams join %1$s to accept"), BEEN_INVITED("%1$s has invited you to a team."), NOT_IN_TEAM("You are not in a team."), TEAM_CREATED("Team Created, you can now invite other players."), TEAM_MEMBER("Team members: ");
```
Comment (RobertSkalko) [inline src/main/java/com/robertx22/age_of_exile/uncommon/localization/Command.java]: can you please put each of these on new lines lol. At least on my github view it looks nearly all stacked together

File: src/main/java/com/robertx22/age_of_exile/gui/texts/textblocks/SalvageBlock.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class SalvageBlock extends AbstractTextBlock{
+
+    private final ISalvagable data;
+
+    public SalvageBlock(ISalvagable data) {
+        this.data = data;
```
Comment (RobertSkalko) [inline src/main/java/com/robertx22/age_of_exile/gui/texts/textblocks/SalvageBlock.java:21]: This block should only show if shift or control are pressed

Comment (TUsama) [reply src/main/java/com/robertx22/age_of_exile/gui/texts/textblocks/SalvageBlock.java:21]: ok

**Comment by TUsama** [comment]
Robert I've changed the way how to enable the dev tool, I think maybe you will interested in:

https://github.com/TUsama/Mine-And-Slash-Rework/commit/8d5691218dbf664d9135cd006fa1e503914ed793

with this change, you don't need to recompile the whole mod to disable the dev tool. The dev tool is enabled by default if you use runClient to run Minecraft. For the normal player, if they also want to enable the dev tool, they can simply add -Dmas.dev.tool=true to the JVM argument.

if you think this change is ok enough I can cherry-pick it to this branch.

**Comment by TUsama** [comment]
Robert I think this PR is ready to be reviewed, it's big enough lmao

**Comment by RobertSkalko** [comment]
Seems alright

**Pull request merged**